TShockAPI/Commands.cs

```diff
@@ -2763,14 +2798,14 @@ private static void TP(CommandArgs args)
 					var target = players[0];
 					if (!target.TPAllow && !args.Player.HasPermission(Permissions.tpoverride))
 					{
-						args.Player.SendErrorMessage("{0} has disabled players from teleporting.", target.Name);
+						args.Player.SendErrorMessage($"{target.Name} has disabled players from teleporting to {(target.TPlayer.Male ? "him" : "her")}");
```

Comment:
```suggestion
						args.Player.SendErrorMessage($"{target.Name} has disabled players from teleporting to them.");
```

Gender neutrality is probably better here